cosmwasm/contracts/ntt-global-accountant/src/structs/relayer.rs

```diff
@@ -0,0 +1,273 @@
+            0x07, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x8f,
+            0x26, 0xa0, 0x02, 0x5d, 0xcc, 0xc6, 0xcf, 0xc0, 0x7a, 0x7d, 0x38, 0x75, 0x62, 0x80,
+            0xa1, 0x0e, 0x29, 0x5a, 0xd7, 0x00, 0x4f, 0x99, 0x4e, 0x54, 0x54, 0x08, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x03, 0xe8, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0xa8, 0x80, 0x85, 0xe6, 0x37, 0x0a, 0x55, 0x1c, 0xc0, 0x46,
+            0xfb, 0x6b, 0x1e, 0x3f, 0xb9, 0xbe, 0x23, 0xac, 0x3a, 0x21, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x8f, 0x26, 0xa0, 0x02, 0x5d, 0xcc,
+            0xc6, 0xcf, 0xc0, 0x7a, 0x7d, 0x38, 0x75, 0x62, 0x80, 0xa1, 0x0e, 0x29, 0x5a, 0xd7,
+            0x27, 0x12, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x60,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x07, 0xa1, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x04, 0x6f, 0x53, 0x99, 0xe7, 0x27, 0x12,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x7a, 0x0a, 0x53, 0x84, 0x77, 0x76, 0xf7, 0xe9, 0x4c, 0xc3, 0x57, 0x42,
+            0x97, 0x1a, 0xcb, 0x22, 0x17, 0xb0, 0xdb, 0x81, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x7a, 0x0a, 0x53, 0x84, 0x77, 0x76, 0xf7, 0xe9,
+            0x4c, 0xc3, 0x57, 0x42, 0x97, 0x1a, 0xcb, 0x22, 0x17, 0xb0, 0xdb, 0x81, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xe4, 0x93, 0xcc, 0x4f,
+            0x06, 0x98, 0x21, 0x40, 0x4d, 0x27, 0x2b, 0x99, 0x4b, 0xb8, 0x0b, 0x1b, 0xa1, 0x63,
+            0x19, 0x14, 0x00,
+        ];
+        let delivery = DeliveryInstruction::deserialize(&payload).unwrap();
+
+        assert_eq!(delivery.target_chain, 10002);
```

Comment:
FYI: If `DeliveryInstruction` derives `PartialEq` and `Eq`, we could just assert that your `delivery` struct is equal to an expected struct. It's nicer to do it this way in case the test inadvertently misses checking a struct field.

Reply:
great idea, thanks. done!